components/scripts/lib/info.sh

```diff
@@ -172,6 +174,35 @@ print_realized_build_time_savings() {
   fi
 }
 
+print_potential_build_time_savings() {
```

Comment:
As this is currently written, the `Potential build time savings` is always displayed. In cases where no cacheable tasks were executed, the metrics are identical to `Realized build time savings`. For example:

![image](https://user-images.githubusercontent.com/5797900/210608028-243ee366-d604-4922-a15f-0208c2eb7d46.png)

I wonder if it makes sense to conditionally print `Potential build time savings` only when the number of executed cacheable tasks is > 0?

Reply:
I think it is better to always display it. I think it would create more confusion to only display it if there were any unrealized build time savings (it would lead users to have to ask themselves why potential build time savings weren't displayed).

Reply:
Resolving because we came up with a better way to display this which we will address in a future PR.

Reply:
We had a longer discussion about this and came up with a proposal that will be implemented in a separate PR.